js/customize-image-gallery-control.js

```diff
@@ -51,20 +62,79 @@
             this.frame.open();
         },
 
+        /**
+         * Initiate the media modal select frame.
+         * Save it for using later in case needed.
+         */
         initFrame: function() {
 
+            var control = this, preSelectImages;
+
             this.frame = wp.media({
                 button: {
-                    text: 'Select'
+                    text: control.params.labels.select
                 },
                 states: [
                     new wp.media.controller.Library({
-                        title: 'Select gallery images',
+                        title: control.params.labels.title,
                         library: wp.media.query({ type: 'image' }),
                         multiple: 'add'
                     })
                 ]
             });
+
+            /**
+             * Pre-select images according to saved settings.
+             */
+            preSelectImages = function() {
+                var selection, ids, attachment;
+                selection = control.frame.state().get( 'selection' );
+                ids = control.setting.get();
+                ids.forEach( function( id ) {
```

Comment:
Use `_.each( ids, ... )` instead of `ids.forEach( ... )`, as the latter is not available in IE8.